faster_whisper/transcribe.py

```diff
@@ -318,8 +318,11 @@ def transcribe(
         self.logger.info(
             "Processing audio with duration %s", format_timestamp(duration)
         )
-
-        if vad_filter:
+        if isinstance(vad_filter, list):
```

Comment:
Given that you expect an iterable, can we generalize this test to support any kinds of iterable object?